install/Makefile

```diff
@@ -6,34 +6,46 @@ RELEASE_VERSION ?= latest
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/046-Crd-kafkabridge.yaml metadata.labels.component "kafkabridges.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/047-Crd-kafkaconnector.yaml metadata.labels.component "kafkaconnectors.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/048-Crd-kafkamirrormaker2.yaml metadata.labels.component "kafkamirrormaker2.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/049-Crd-kafkarebalance.yaml metadata.labels.component "kafkarebalances.kafka.strimzi.io-crd"
-
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.release "{{ .Release.Name }}" \; && cd $$oldpath
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.heritage "{{ .Release.Service }}" \; && cd $$oldpath
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec $(SED) -i -e '1s/^/{{- if .Values.createGlobalResources -}}\n/' -e '$$s/$$/\n{{- end -}}/' {} \; && cd $$oldpath
+	for HELM_VERSION in 2 3	; do	\
+		HELM_CHART_SUBFOLDER=helm$$HELM_VERSION	;	\
+		oldpath=`pwd`	;	\
+		$(CP) ./cluster-operator/043-Crd-kafkatopic.yaml ./topic-operator/04-Crd-kafkatopic.yaml	;	\
+		$(CP) ./cluster-operator/044-Crd-kafkauser.yaml ./user-operator/04-Crd-kafkauser.yaml	;	\
+		$(CP) ./cluster-operator/040-Crd-kafka.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/040-Crd-kafka.yaml	;	\
+		$(CP) ./cluster-operator/041-Crd-kafkaconnect.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/041-Crd-kafkaconnect.yaml	;	\
+		$(CP) ./cluster-operator/042-Crd-kafkaconnects2i.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/042-Crd-kafkaconnects2i.yaml	;	\
+		$(CP) ./cluster-operator/043-Crd-kafkatopic.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/043-Crd-kafkatopic.yaml	;	\
+		$(CP) ./cluster-operator/044-Crd-kafkauser.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/044-Crd-kafkauser.yaml	;	\
+		$(CP) ./cluster-operator/045-Crd-kafkamirrormaker.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/045-Crd-kafkamirrormaker.yaml	;	\
+		$(CP) ./cluster-operator/046-Crd-kafkabridge.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/046-Crd-kafkabridge.yaml	;	\
+		$(CP) ./cluster-operator/047-Crd-kafkaconnector.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/047-Crd-kafkaconnector.yaml	;	\
+		$(CP) ./cluster-operator/048-Crd-kafkamirrormaker2.yaml ../helm-charts/$$HELM_CHART_SUBFOLDER/strimzi-kafka-operator/templates/048-Crd-kafkamirrormaker2.yaml	;	\
```

Comment:
I think here you are missing the 049 file with the KafkaRebalance resource.

Reply:
I have rebased to the latest and, you are right I missed it...